README.md

```diff
@@ -309,8 +316,8 @@ First source Xilinx Vivado 2020.1
 ```
 ipbb init <project name>
 cd <project name>
-ipbb add git ssh://git@gitlab.cern.ch:7999/p2-xware/firmware/emp-fwk.git -b v0.7.3
-ipbb add git https://github.com/apollo-lhc/CM_FPGA_FW -b v1.2.2
+ipbb add git ssh://git@gitlab.cern.ch:7999/p2-xware/firmware/emp-fwk.git -b v0.8.0
+ipbb add git https://github.com/apollo-lhc/CM_FPGA_FW -b v2.2.1
```

Comment:
these are obsolete, for the record. They should be 0.9.0-pre2 and v3.0.0 for emp-fwk and CM_FPGA_FW